stage #0/module #4 Algorithms, Data Structures/README.md

```diff
@@ -0,0 +1,28 @@
+<h1 align="center">Algorithms. Data structures</h1>
+
+<p align="center"><img src ="media/algorithm_logo.jpg" alt="Algorithm Logo" width = 30%></p>
```

Comment:
In most of materials of https://github.com/mjc-school/MJC-School, no HTML markup language is used, only "Markdown" markup language. Could you please convert HTML ones into MD?